src/client/VZCodeContext.tsx

```diff
@@ -355,6 +357,9 @@ export const VZCodeProvider = ({
   const [hoveredItemId, setHoveredItemId] =
     useState<ItemId | null>(null);
 
+  // Implement the saving/saved status
+  const [savingStatus, setSavingStatus] = useState('All changes saved');
```

Comment:
```suggestion
  const savingStatus = useSavingStatus({shareDBDoc});
```

where `useSavingStatus` is pretty much the same as `usePending` from the old example.